panda/src/framework/windowFramework.cxx

```diff
@@ -323,6 +323,31 @@ get_aspect_2d() {
     }
 
     _aspect_2d.set_scale(1.0f / this_aspect_ratio, 1.0f, 1.0f);
+
+    // Create child nodes of aspect_2d anchored to edges/corners of the window.
+    _a2d_top_center = _aspect_2d.attach_new_node("a2d_top_center");
+    _a2d_bottom_center = _aspect_2d.attach_new_node("_a2d_bottom_center");
+    _a2d_left_center = _aspect_2d.attach_new_node("_a2d_left_center");
+    _a2d_right_center = _aspect_2d.attach_new_node("_a2d_right_center");
+    _a2d_top_left = _aspect_2d.attach_new_node("_a2d_top_left");
+    _a2d_top_right = _aspect_2d.attach_new_node("_a2d_top_right");
+    _a2d_bottom_left = _aspect_2d.attach_new_node("_a2d_bottom_left");
+    _a2d_bottom_right = _aspect_2d.attach_new_node("_a2d_bottom_right");
```

Comment:
Some of these node names have _ suffix some don't

Let's just give them the same names as in ShowBase, so a2dTopCenter etc, so:
```python
_a2d_bottom_right = _aspect_2d.attach_new_node("a2dBottomRight");
```